ros_buildfarm/config/release_build_file.py

```diff
@@ -43,6 +43,12 @@ def __init__(self, name, data):
             self.abi_incompatibility_assumed = \
                 bool(data['abi_incompatibility_assumed'])
 
+        self.build_environment_variables = None
+        if 'build_environment_variables' in data:
+            self.build_environment_variables = \
+                    data['build_environment_variables']
+            assert(isinstance(self.build_environment_variables, dict))
```

Comment:
Please add the new option to the documentation.

Reply:
First pass at documentation in 045cf5e.